apps/desktop-wallet/src/features/transactionsDisplay/transactionDetailsModal/AddressesDataRows.tsx

```diff
@@ -24,20 +25,24 @@ import AddressBadge from '@/components/AddressBadge'
 import DataList from '@/components/DataList'
 import HashEllipsed from '@/components/HashEllipsed'
 import IOList from '@/components/IOList'
-import { TransactionDetailsModalSectionProps } from '@/features/transactionsDisplay/transactionDetailsModal/types'
+import { selectPendingSentTransactionByHash } from '@/features/sentTransactions/sentTransactionsSelectors'
+import PendingSentAddressBadge from '@/features/transactionsDisplay/transactionDetailsModal/PendingSentAddressBadge'
+import { TransactionDetailsModalTxProps } from '@/features/transactionsDisplay/transactionDetailsModal/types'
 import useOnAddressClick from '@/features/transactionsDisplay/transactionDetailsModal/useOnAddressClick'
 import useOpenTxInExplorer from '@/features/transactionsDisplay/transactionDetailsModal/useOpenTxInExplorer'
 import useTransactionDirection from '@/features/transactionsDisplay/useTransactionDirection'
+import { useAppSelector } from '@/hooks/redux'
 
-const AddressesDataRows = ({ tx, addressHash }: TransactionDetailsModalSectionProps) => {
+const AddressesDataRows = ({ tx, refAddressHash }: TransactionDetailsModalTxProps) => {
```

Comment:
I renamed `addressHash` to `refAddressHash` because in the context of a "transaction" we always want to know "which address as reference" do we want to display info with? Displaying a "moved" transaction in the context of a wallet will appear as "incoming" in the context of an address. So, `refAddressHash`.

Reply:
Makes sense!